README.md

```diff
@@ -36,6 +36,12 @@ However, this isn't meant to replace any validation or mask library, you should
 npm install --save cleave.js
 ```
 
+#### CDN
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/cleave.js@1/dist/cleave.min.js"></script>
+```
+
```

Comment:
maybe try not to include the script in our documentation?
instead, add something like:

cleave.js is available on [cdnjs.com](https://cdnjs.com/libraries/cleave.js) and on [jsDelivr](https://www.jsdelivr.com/package/npm/cleave.js)

thoughts?